.github/workflows/build.yml

```diff
@@ -216,3 +216,26 @@ jobs:
       - name: Run default clippy linting
         run: |
           cargo clippy -- -Aclippy::erasing_op -Aclippy::never_loop -Aclippy::if_same_then_else -Dclippy::try_err
+
+  mutation:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request'
```

Comment:
I think it does not need to run it for every PR. PRs that do not change any relevant code could be skipped (e.g. upgrading dependencies, test changes, etc?). Not sure how to deal with it in this project.

Reply:
I do believe it would spend a lot of time in PRs like https://github.com/lightningdevkit/rust-lightning/pull/3063, for example. It's what is expected?